### ISSUE TITLE
default channel trigger doesn't work for txt files

### ISSUE THREAD

**Issue by vinferrer** [physiopy/phys2bids#83]
Hi, doing some testing , i have realized that the default of  -chtrig is not vaild for labchart txt files since the trigger channel happens to be index 1 no index 0, should we change default or only correct it when txt files are used?

**Comment by smoia** [comment]
We don't know yet if all the txt files have the trigger in position 1. Actually, we might expect it to be all over the place.

A better approach would be to force the user to put it, *if* the option `-info` is not present.
For instance, we can set its default to `None`, and raise an exception if `info` is not True and `chtrig` is None.

Another approach would be to *not* consider the first channel when processing labchart files - so that we can propose to standardise the trigger as first channel during acquisition, and not break the code.

What do you think @RayStick @eurunuela @rmarkello  (and anyone else with an opinion in the matter, ofc)?

**Comment by RayStick** [comment]
Yes, I noticed this as a problem too. In our labchart txt files the first column [0] is time, and the second column is the trigger [1]. In these txt files, the first column will always be time, but the second column won't always be trigger. It normally is, but if depends completely on how the user has set up their hardware.

**Comment by RayStick** [comment]
As to the solution, I think either of the things you suggest would work @smoia. Forcing the user to input it may be best, at least for now.

**Comment by RayStick** [comment]
I've noticed that we have some Chicago data labelled 'multiscan' and some labelled ' 2minrest' so best to test on the '2minrest' for now. 

I have also put a bit more Chicago data up on OSF (labelled ‘Test2’) in case it helps with this testing. One scan exported from lab chart in 4 different ways: header with time column, header with no time column, no header with time column, no header with no time column. I have included the number of TRs in the file name, so you know if phys2bids is getting it right.
